Microsoft.Dynamics365.UIAutomation.Api.UCI/Elements/Dialog.cs

```diff
@@ -190,5 +190,28 @@ public bool SetStateDialog(bool clickOkButton)
         {
             return _client.SetStateDialog(clickOkButton);
         }
+
+        /// <summary>
+        /// Clicks on entity dialog ribbon button
+        /// </summary>
+        /// <param name="buttonname"></param>
+        /// <param name="subButtonName"></param>
+        /// <param name="secondSubButtonName"></param>
+        /// <returns></returns>
+        public bool ClickCommand(string buttonName, string subButtonName = null, string secondSubButtonName = null)
+        {
+            return _client.ClickCommand(buttonName, subButtonName, secondSubButtonName);
+        }
+
+        /// <summary>
+        /// Clicks on entity dialog ribbon button

```

Comment:
nit: Can we update the summary info here

Reply:
done